doc/nrf/known_issues.rst

```diff
@@ -474,29 +458,29 @@ Immutable bootloader board restrictions
 Build system
 ============
 
-.. rst-class:: v1-4-99-dev1 v1-4-2 v1-4-1 v1-4-0
+.. rst-class:: v1-4-2 v1-4-1 v1-4-0
 
 NCSDK-6898: Overriding child images
   Adding child image overlay from the :file:`CMakeLists.txt` top-level file located in the :file:`samples` directory overrides the existing child image overlay.
 
   **Workaround:** Apply the configuration from the overlay to the child image manually.
 
-.. rst-class:: v1-4-99-dev1 v1-4-2 v1-4-1 v1-4-0
+.. rst-class:: v1-4-2 v1-4-1 v1-4-0
 
 NCSDK-6777: Project out of date when :option:`CONFIG_SECURE_BOOT` is set
   The DFU :file:`.zip` file is regenerated even when no changes are made to the files it depends on.
   As a consequence, SES displays a "Project out of date" message even when the project is not out of date.
 
   **Workaround:** Apply the fix from `sdk-nrf PR #3241 <https://github.com/nrfconnect/sdk-nrf/pull/3241>`_.
 
-.. rst-class:: v1-4-99-dev1 v1-4-2 v1-4-1 v1-4-0
+.. rst-class:: v1-5-0 v1-4-2 v1-4-1 v1-4-0
```

Comment:
This is fixed by https://github.com/nrfconnect/sdk-nrf/pull/3910 which will be included in the relase, so I guess we can safely remove this.

```suggestion
.. rst-class:: v1-4-2 v1-4-1 v1-4-0
```